packages/providers/inpage-providers-hub/src/floatingButton/index.tsx

```
@@ -0,0 +1,746 @@
+                fontSize: '11.2px',
+              }}
+            >
+              {i18n.maliciousSiteWarning}
+            </span>
+            <svg
+              width="18"
+              height="16"
+              viewBox="0 0 18 16"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)